components/HostsWithData.js

```diff
@@ -71,52 +72,29 @@ class HostsWithData extends React.Component {
 
     const limit = this.props.limit || COLLECTIVE_CARDS_PER_PAGE * 2;
     return (
-      <div className="HostsContainer" ref={node => (this.node = node)}>
-        <style jsx>
-          {`
-            :global(.loadMoreBtn) {
-              margin: 1rem;
-              text-align: center;
-            }
-            .filter {
-              width: 100%;
-              max-width: 400px;
-              margin: 0 auto;
-            }
-            :global(.filterBtnGroup) {
-              width: 100%;
-            }
-            :global(.filterBtn) {
-              width: 33%;
-            }
-            .Hosts {
-              display: flex;
-              flex-wrap: wrap;
-              flex-direction: row;
-              justify-content: center;
-              overflow: hidden;
-              margin: 1rem 0;
-            }
-            .HostsContainer :global(.CollectiveCard) {
-              margin: 1rem;
-            }
-          `}
-        </style>
-
-        <div className="Hosts cardsList">
+      <Container className="HostsContainer" ref={node => (this.node = node)}>
+        <Container
+          className="cardsList"
```

Comment:
We should now be able to remove these ones too 

```suggestion
      <Container ref={node => (this.node = node)}>
        <Container
```

Reply:
Done 👍🏽